examples/booking_app/lib/feature_hotel/search/views/search_page.dart

```diff
@@ -21,6 +20,8 @@ class SearchPage extends StatefulWidget {
 class _SearchPageState extends State<SearchPage> with TickerProviderStateMixin {
   late AnimationController animationController;
   final ScrollController _scrollController = ScrollController();
+  DateTimeRange? dateRange;
```

Comment:
@DDavidPrime it's not recommended to keep a state within the UI Layer. Please have a look at the comment above.